bcgov/biohubbc
# SIMSBIOHUB-57: Taxonomy text display

# Overview

## Links to Jira tickets

[SIMSBIOHUB-57](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-57)

## Description of relevant changes

- updated column to `Focal Species`
- Updated taxonomy service to only return English name and scientific name

## Testing Steps
 1. Navigate to/ Create a project
 2. Edit/ Create a Survey
 3. Search for and add focal and ancillary species (see names are English and Scientific only)
 4. Navigate back to project details page 
 5. See Survey list `Focal Species` column
 6. See species are English and Scientific name only

Important: If an English name doesn't exist for a particular species, only the scientific name will be visible

## PR Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### Code


### Style


### Documentation


### Tests


### Linting/Formatting

       _See the `lint` commands in package.json_
       _See the `format` commands in package.json_

### SonarCloud
